tests/integration/test_charm.py

```diff
@@ -82,7 +82,7 @@ async def test_add_profile_relation(ops_test: OpsTest):
         [PROFILES_CHARM_NAME, CHARM_NAME],
         status="active",
         raise_on_error=True,
-        timeout=300,
+        timeout=600,
```

Comment:
What would be the reason for increasing the timeout?

Reply:
the charm is taking more time to reach active, it was failing because of timeout in [this run](https://github.com/canonical/kubeflow-dashboard-operator/actions/runs/6171092792/job/16748657254)
not sure if this is a transient CI issue or related to the update